clone-flexdll

```diff
@@ -8,9 +8,6 @@ if [ -d "flexdll" ] && [ -f "flexdll/flexdll.c" ]; then
     echo "[Flexdll] Already present, no need to clone."
 else
     echo "[Flexdll] Cloning..."
-    git clone https://github.com/alainfrisch/flexdll
-    cd flexdll
-    git checkout 0.39
-    cd ..
+    git clone https://github.com/alainfrisch/flexdll --branch 0.39
```

Comment:
since 0.39 is also a tag we don't need to cd

Reply:
:+1: